combat-trainer.lic

```diff
@@ -2259,16 +2261,42 @@ class AttackProcess
       game_state.loaded = false
       waitrt?
     elsif game_state.loaded
-      actions = game_state.next_aim_action.split(';')
-      actions.each do |action|
-        first_word = action.split(' ')[0].downcase
-        matches = [/^You /]
-        matches = ['You get', 'You draw out', 'You pick up', "You're already holding that"] if @get_actions.include?(first_word)
-        matches = ['Roundtime',"You aren't close enough to attack"] if @rt_actions.include?(first_word)
-        matches = ['You put', 'You sheathe'] if @stow_actions.include?(first_word)
-        bput(action.strip, matches)
-        pause 0.25
-        waitrt?
+      if game_state.aiming_trainables
```

Comment:
💡 You could remove a level of indentation by combining lines 2263 and 2264 into `elsif game_state.loaded && game_state.aiming_trainables` and then line 2288 becomes ``elsif game_state.loaded`.

Reply:
I was going to do this, but then figured we'd deprecate the other code. Then realized it has it's uses, so left it in. I did the suggested.